internal/starlark/gettransformdata/simpletransform.go

```diff
@@ -43,17 +44,10 @@ def lower_number_of_replicas(x):
     return x
 
 def change_the_ports(x):
-    x["spec"]["template"]["spec"]["containers"][0]["ports"] = answers("services.svc1.ports")
+    x["spec"]["template"]["spec"]["containers"][0]["ports"] = query("services.svc1.ports")
```

Comment:
Do we want to change the query parameters?

Reply:
done